### ISSUE TITLE
Daniel Kerr & OpenCart.Com ONGOING SCAM

### ISSUE THREAD

**Issue by manakill** [opencart/opencart#2915]
I am the President of Explicit Clothing Incorporated, Licensed and Registered in NV.
Upon purchasing a "PayPal Payments Advanced OCMod" the system put my order on hold.
Then it said it was flagged as fraudulent activity. I have heard other horror stories about OpenCart.com provided extensions and how you should go elsewhere.. I didn't listen..

I don't know what kind of BS security system they are running there but this is a CORPORATE PAYPAL ACCOUNT, CORPORATE EMAIL ACCOUNT, PAID IN FULL.
If PayPal's security checks say payment is good, then F'in-A

I see now that "Daniel Kerr" is the operator of OpenCart.com and is also the operator of this GitHub Repository. Interesting scam you are running here Daniel.. They say below that if you report them, they will BAN you in return also.. Pretty sure PayPal doesn't agree with this customer service policy.

***STEPS YOU CAN TAKE***
Report and flag this OpenCart charge as FRAUD or UNAUTHORIZED CHARGE with PayPal and your Credit Card company.
You can report the scam to BBB.com, FBI Internet Fraud, StopFraud.gov, OnGaurdOnline.gov
You can also Call and Email PayPal to complain about the seller OpenCart.com(OpenCart LTD, Daniel Kerr) - Merchant account providers don't like when customers are getting billed for items and not delivering.

![image](https://cloud.githubusercontent.com/assets/9597295/7165605/97fe89c2-e376-11e4-8914-92ff5d8c13fd.png)

![image](https://cloud.githubusercontent.com/assets/9597295/7165663/ede52d46-e376-11e4-94e7-bb7031a20d5d.png)

![image](https://cloud.githubusercontent.com/assets/9597295/7165666/f6780a96-e376-11e4-9bd5-e26f9a180894.png)

**Comment by EveryAuction** [comment]
> I have heard other horror stories about OpenCart.com provided extensions and how you should go elsewhere.. <
Opencart does not provide Extensions, it just handles the financial transaction betwenn der Seller and the Buiyer of such Things.. Just to make sure about this. Anything else cannot be judged, by Outsiders.

**Comment by manakill** [comment]
You are apparently missing the point. It is a marketplace. You pay them and the money goes to OpenCart LTD, NOT the seller directly. Downloads are provided directly from OpenCart.com

**Comment by beifloyd** [comment]
Calm down, it's likely the security check is performed by a third party company specialised in online fraud.
Have you tried to contact the company to solve the issue before coming here. The GitHub account is for Opencart code which is open source. 
So, the point here is:
- either you are a scam
- or your account did not tick all the check boxes when processed automatically. If that's the case I'm sure they'll solve this for you if you make contact with Opencart.

**Comment by manakill** [comment]
Yes, called the "Hong Kong" number, nobody answers. Emailed, no answer, opened tickets, no answer.

I know what GitHub is, the reason I'm here is DANIEL KERR is the ringleader here, and the ringleader performing illegal activities.

I have used OpenCart.com in the past with no issues, the past year has been nothing but problems.

***USE 3RD PARTY VENDOR SITES, NOT OPENCART.COM TO PURCHASE EXTENSIONS***

Again, payment was just fine. The issue here is the illegal receipt of funds under false pretense and failure to deliver.

**Comment by jamesallsup** [comment]
This is nothing more than a rant about an order getting blocked by a security tool used to vet out 99% of other fraud. Since this was created yesterday and your transaction was on 15th that is hardly enough time for any replies from the support staff.

No one is SCAMMING ANYONE!! Anti fraud protection systems are used to protect our sellers from losing their sales by nasty people using the PayPal dispute service for fun. If you had just contacted support first, waited, then one of the support team in HK will have helped you with your issue.

Now instead, all you have done is annoy people with your "scam" comments and continued thread posts that have spammed my inbox.

The issue will now be LOCKED! If you create another issue or update any others on this subject then your account will be blocked from GutHub too.

1. POLITELY contact the support team via the website, explain if you are wanting a refund or the module still.
2. Wait patiently, they get a lot of enquirers so may be busy with other people.

Easy.

**Comment by danielkerr** [comment]
manakill your first order had a high fraud score. rather than provide ID wich you where s=asked for you told me:

Username: manakill 
Name: Rich Comiskey 
E-Mail: manakill@gmail.com 
IP: 96.25.246.88

Enquiry:

Just refund both of them. This is crazy. Refund the PayPal payment, and also the CC Payment. Else I'll do chargebacks on both.

so i refund you. i don't kindly to threats. i prefer not to do business with people like you.

again you tried to buy from me using different details and refused to send id. then you opened  2 other accounts to buy from two more times with different information. both times my system flagged your order again.

I have addresses from 4 states in the US from you. 

Nevada
Michigan
wyoming
North Carolina

all flagged as fraud by maxminds anti fraud system.

you have only emailed me once from the 1st transactions requesting a refund. the 2nd one i refunded myself as you were told you were banned.

im not into playing cat an mouse games of you trying to order from me and hoping my system will catch you everytime. your orders were set to a fraud status. i was waiting for your email to email me to request a refund so  could warn you not to try to order again. 

you have not emailed me requesting  a refund on your last 2 transactions.

you did start a dispute on paypal. so i havern't ripped you, as its still possible to refund you.

but how are you gong to get the message to stop trying to purchase from me? i dont want to do business with you!

**Comment by manakill** [comment]
Daniel Kerr, 
Has it occurred to you that I set up systems for business owners? I've set up 8 for 8 different people on 8 different domains not including my own.

NONE of MY CLIENTS want to provide ID to a Hong Kong based business that immediately flags someone legitimate. In all my online dealings, your system seems to be the most shady as far as it goes. PayPal has no problem with me or my dealings. Payment goes through fine. I've never issued a chargeback unless it was not delivered, all of my providers can back me up as I have Valid Licenses with them. I am fully legit on State and Federal level.

I am still here, still buying from other great sellers. I'm not the fraudster here, you sir are.

Damn me for trying you again!
I should have learned from all the times you burned me in the past. You are right!

***MORAL OF THE STORY - DON'T BUY FROM DANIEL KERR AND OPENCART.COM***

**Comment by danielkerr** [comment]
@EveryAuction opencart does over 400 orders a day and has been doing this for a couple of years now. the people who i have had issue with are less than a a faction of a percent.

this guy here:

http://opencart.pissedconsumer.com/opencart-shopping-cart-owner-daniel-kerr-ripped-me-off-20130602414068.html

started a dispute and got a refund from paypal on 2 extensions he was using on his web site. hes still using them today. i can post links if you like. what did he expect to happen. he lied to paypal to get a refund. he then kept coming back to buy again under different accounts. 

look on the comments. 

from "Scam bad service", this guy is from Burma, who used VCC to buy from me using a US address. told him not to attempt to buy from us using VCC again. i refunded him but still posts that hes been ripped off.

look at comments from Chucky. look at his last one apologizing. he started posting comments against me because he had a problem with an extension, and because my staff did not answer his question on a Saturday he posted on pissedconsumer.

**Comment by manakill** [comment]
You are right, I can't do anything but rage and report you places.

But if your staff handled the fraction of a percent, properly, you wouldn't have raging mad customers, am I right?

***At least re-evaluate your system and make it easier to "verify ID" without asking for SENSITIVE INFORMATION.***

You can't blame me for the risks on online dealings, I face them everyday. 
***Banning a paid customer without just cause is plain wrong.***

**Comment by danielkerr** [comment]
@manakill  so you were using your clients credit card details! credit card details from different states. this is why you were flagged as fraud!

I asked for ID to protect my self in case the transactions come back as fraud. i used to get 30 transactions a day by people from vietnam using proxies and stolen card info. now i get about 1 a month since using maxminds anti fraud system and requesting ID off buyers with high fraud scores.

you should use your own credit card and invoice the client or ask the client to purchase from me and email you the files.

all you had to do was provide the ID to me one time and you would be added to a safe list. you don't even have to provide the full ID, many people blank out parts of the ID. I just need something that looks official. 

i must have 20 fraud flagged transactions a day. out of the 20 at least 15 provide ID no problem. the remaining 5 dont get the ID request because they put fake emails down. if ID has not been provided the transaction get refunded in 24 hours.

"If PayPal's security checks say payment is good, then F'in-A"

this shows you inexperience selling online.

I can tell you know i have seen files with 10,000's of peoples hacked paypal email and passwords after   some one in the community hacked a veitnamese fraudster site. paypal was contacted and the people whose accounts were on the list were alerted. 

you don't listen the first time to what you are told and think you are being clever by just opening a new account.  

if you are running a business using a companies software that has been provided to you for free logic dictates you should try to have a good business relationship with that company and not post threats or
calling them scammers!

your account remains banned! 

good bye!

**Comment by kombi747** [comment]
To anyone reading this post... There some good advice here on how to buy software for clients you work with. 

If you area contractor either buy the software under your company name and invoice the client or work with your client to create the needed account and have them buy...
Honestly IT has changed and if you want to be straight up with a client have them buy any software, domain name or hosting.  Under their name.

I have helped 100’s of businesses clean up the simple avoidable mess of not knowing how to access their IT stuff (software, domain name, hosting, website, and backend e-mail).  As a contractor you own your client the ability to not worry about not having all the proper documentation an password to their IT stuff (Pretty big list)... IT folks come and go but hopefully there business will be around for a while.

So Business owner’s and contractors be proactive and document all your IT information. Do not have a single point of failure or what I call the dead man factor in your org.  Be active stay active.  You do not need to know IT to have all documentation you need. You just have to be a good manager/owner.  

To end get a password safe such as http://keepass.info/ and note all need information in the note fields.  Such as contract #’s support #”s and e-mail etc…You will be happy when you cannot find the contractor or e-mail when you need to upgrade, down grade, pay an invoice or get support.  
A little off topic but there get to be a moral to this story or it just TV/Web drama and a waste of time.
